src/backend/vulkan/src/device.rs

```diff
@@ -1891,11 +1891,31 @@ impl d::Device<B> for super::Device {
     }
 
     fn start_capture(&self) {
-        //TODO: RenderDoc
+        #[cfg(feature = "renderdoc")]
+        unsafe {
+            match self.shared.instance.render_doc_entry.as_ref() {
+                Ok(entry) => {
+                    ((*entry.api).StartFrameCapture.unwrap())(ptr::null_mut(), ptr::null_mut());
+                }
+                Err(cause) => warn!("Could not start render doc frame capture: {}", cause),
+            };
+        }
+        #[cfg(not(feature = "renderdoc"))]
+        debug!("RenderDoc frame capture only supported with feature 'renderdoc' and 'debug_assertions'");
```

Comment:
they are going to see the `warn!` in this case, so this extra message seems redundant

Reply:
removed by the feature removal as well